packages/db/tests/models/test-collection-model.js

```diff
@@ -76,6 +80,149 @@ test('CollectionPgModel.upsert() overwrites a collection record', async (t) => {
     {
       ...updatedCollection,
       files: JSON.parse(updatedCollection.files),
+      meta: JSON.parse(collectionRecord.meta),
     }
   );
 });
+
+test('Collection.searchWithUpdatedAtRange() returns an array of records if no date range specified', async (t) => {
+  const {
+    knex,
+    collectionPgModel,
+  } = t.context;
+
+  const collectionName = cryptoRandomString({ length: 10 });
+  const records = times(3, (i) => fakeCollectionRecordFactory({
+    name: collectionName,
+    version: i,
+  }));
+  await Promise.all(records.map((r) => collectionPgModel.create(knex, r)));
+
+  const searchResponse = await collectionPgModel.searchWithUpdatedAtRange(
+    knex,
+    { name: collectionName },
+    {}
+  );
+
+  t.is(searchResponse.length, 3);
+});
+
+test('Collection.searchWithUpdatedAtRange() returns a filtered array of records if a date range is specified', async (t) => {
+  const {
+    knex,
+    collectionPgModel,
+  } = t.context;
+
+  const collectionName = cryptoRandomString({ length: 10 });
+  const records = times(3, (i) => fakeCollectionRecordFactory({
```

Comment:
Can we initialize a date explicitly on these records to make it clear why they wouldn't be returned for the requested date range?

Reply:
Yes: f027ff901